cli/internal/cmd/upgradecheck.go

```diff
@@ -47,7 +47,7 @@ func newUpgradeCheckCmd() *cobra.Command {
 		RunE:  runUpgradeCheck,
 	}
 
-	cmd.Flags().BoolP("write-config", "w", false, "update the specified config file with the suggested versions")
+	cmd.Flags().BoolP("update-config", "u", false, "update the specified config file with the suggested versions")
```

Comment:
I'm unsure if we should go for consistency and rename it, or go for UI compatibility and keep it as it was. What's your opinion, @derpsteb ?
If we choose to rename it, we'd also need to adapt the docs at workflows/upgrade.md

Reply:
I would go for consistency if your only concern is about the necessary change in the docs.